newIDE/app/src/MainFrame/EditorContainers/HomePage/StoreSection/index.js

```diff
@@ -38,6 +39,17 @@ const StoreSection = ({
     ? searchResults
     : [];
 
+  const existingAssetStoreIds = React.useMemo(
+    () => (project ? enumerateAssetStoreIds(project) : new Set<string>()),
+    [
+      project,
+      // Force recompute existing installed asset ids when the dialog to install them is
+      // opened/closed, so that this list of ids is always up-to-date (but not recomputed at each render,
+      // just when you're likely to need it).
+      isAssetPackDialogInstallOpen,
```

Comment:
flow doesn't complain?
You can probably put an `if (isAssetPackDialogInstallOpen) {...}` in the memo so that we actually rely on this prop

Reply:
Flow no (the typing is valid) but you're right that the hooks linter is complaining. I'm adding this :)